docs/how-to/using-omnitrace-api.rst

```diff
@@ -0,0 +1,294 @@
+.. meta::
+   :description: Omnitrace documentation and reference
+   :keywords: Omnitrace, ROCm, profiler, tracking, visualization, tool, Instinct, accelerator, AMD
+
+****************************************************
+Using the Omnitrace API
+****************************************************
+
+The following example shows how a program can use the Omnitrace API to analyze a program.
```

Comment:
this sentence seems odd
use a program to analyze a program?
perhaps ok

Reply:
I'll reword this to make it clearer.